ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-cpou-upgrade-from-4.14.yaml

```diff
@@ -50,6 +50,12 @@ releases:
       architecture: arm64
       channel: fast
       version: "4.14"
+  source:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
```

Comment:
```suggestion
      version: "4.14"
```

Reply:
same to https://github.com/openshift/release/pull/57024/files#r1772434496